src/Umbraco.Web/Editors/Binders/ContentItemBinder.cs

```diff
@@ -0,0 +1,100 @@
+
+        public ContentItemBinder() : this(Current.Logger, Current.Services, Current.UmbracoContextAccessor)
+        {
+        }
+
+        public ContentItemBinder(ILogger logger, ServiceContext services, IUmbracoContextAccessor umbracoContextAccessor)
+        {
+            _services = services;
+            _modelBinderHelper = new ContentModelBinderHelper();
+        }
+
+        /// <summary>
+        /// Creates the model from the request and binds it to the context
+        /// </summary>
+        /// <param name="actionContext"></param>
+        /// <param name="bindingContext"></param>
+        /// <returns></returns>
+        public bool BindModel(HttpActionContext actionContext, ModelBindingContext bindingContext)
+        {
+            var model = _modelBinderHelper.BindModelFromMultipartRequest<ContentItemSave>(actionContext, bindingContext);
+            if (model == null) return false;
+
+            model.PersistedContent = ContentControllerBase.IsCreatingAction(model.Action) ? CreateNew(model) : GetExisting(model);
+
+            //create the dto from the persisted model
+            if (model.PersistedContent != null)
+            {
+                foreach (var variant in model.Variants)
+                {
+                    if (variant.Culture.IsNullOrWhiteSpace())
+                    {
+                        //map the property dto collection (no culture is passed to the mapping context so it will be invariant)
+                        variant.PropertyCollectionDto = Mapper.Map<ContentPropertyCollectionDto>(model.PersistedContent);
+                    }
+                    else
+                    {
+                        //map the property dto collection with the culture of the current variant
+                        variant.PropertyCollectionDto = Mapper.Map<ContentPropertyCollectionDto>(
+                            model.PersistedContent,
+                            options => options.Items[ResolutionContextExtensions.CultureKey] = variant.Culture);
+                    }
+
+                    //now map all of the saved values to the dto
+                    _modelBinderHelper.MapPropertyValuesFromSaved(variant, variant.PropertyCollectionDto);
+                }
+            }
+
+            return true;
+        }
+
+        private IContent GetExisting(ContentItemSave model)
+        {
+            return _services.ContentService.GetById(model.Id);
+        }
+
+        private IContent CreateNew(ContentItemSave model)
+        {
+            var contentType = _services.ContentTypeService.Get(model.ContentTypeAlias);
+            if (contentType == null)
+            {
+                throw new InvalidOperationException("No content type found with alias " + model.ContentTypeAlias);
+            }
+            return new Content(
+                model.PersistedContent.ContentType.VariesByCulture() ? null : model.Variants.First().Name,
```

Comment:
```ContentType``` is null, I think the one above was supposed to be used.

Reply:
Ah yup! good find, i didn't test creating content. Have pushed the fix